compiler/common-artifacts/CMakeLists.txt

```diff
@@ -33,13 +33,18 @@ set(REQUIREMENTS_FILE "requirements.txt")
 set(REQUIREMENTS_OVERLAY_PATH_TF_2_6_0 "${VIRTUALENV_OVERLAY_TF_2_6_0}/${REQUIREMENTS_FILE}")
 set(REQUIREMENTS_OVERLAY_PATH_TF_2_8_0 "${VIRTUALENV_OVERLAY_TF_2_8_0}/${REQUIREMENTS_FILE}")
 
+set(PYTHON_OVERLAY python3)
+if(PYTHON_EXECUTABLE MATCHES python3.8)
```

Comment:
Ubuntu 18.04 default is python3.6 but we need to use python3.8.
Have you checked this with Ubuntu 18.04 with python3.8 installed but default is python3.8?

Reply:
Maybe it's OK because ubuntu 20.04's default python3 is 3.8. I'll check.

Reply:
there was a typo and fixed to: is ~python3.8~ python3.6?

Reply:
> Have you checked this with Ubuntu 18.04 with python3.8 installed but default is python3.8 python3.6?

I tested on docker `nnfw/one-devtools:bionic` and its default python is 3.6. It's OK.